README.md

```diff
@@ -246,6 +249,7 @@ To read query results containing arrays or maps, pass one of the following struc
 * `trino.NullSliceFloat64`
 * `trino.NullSliceTime`
 * `trino.NullSliceMap`
+* `trino.NullSliceByte`
```

Comment:
Adding `trino.NullSliceByte` here might be misleading, as it's used for `VARBINARY`. This section is about arrays and maps, so `ARRAY(VARBINARY)` will use `trino.NullSlice2Byte`.

Reply:
Add `VARBINARY` to the list in the beginning of this section, maybe after `UUID`.